### DISCUSSION TITLE
Are the config defaults documentated anywhere?

### DISCUSSION THREAD

**Discussion by jnnkB** [nushell/nushell#3221] [Q&A]
Unfortunately I didn't find this information anywhere. Would be nice to know.

**Comment by kubouch** [comment]
Here is a list of possible config options:
https://www.nushell.sh/book/configuration.html#nushell-configuration-with-config-toml

I don't think the defaults are explicitly documented.

**Reply by jnnkB** [reply]
Thanks I already found that. Would be helpful if the defaults could be added.
